client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md

```diff
@@ -6,5 +6,5 @@ superBlock: Data Analysis with Python
 ## Introduction to the How Neural Networks Work Challenges
 
 Neural networks are at the core of what we are calling Artificial Intelligence today. They can seem impenetrable, and even mystical, if you are trying to understand them for the first time, but they don't have to.
-
-Even if you are completely new to neural networks, these lectures by Brandon Rohrer will get you comfortable with the concepts and math behind them.
+Even if you are completely new to neural networks, these lectures by Brandon Rohrer will get you comfortable with the concepts and the math behind them.
+Let us get started. As we walk through the tutorials, you will get a better sense of each concept and gain the required skills.
```

Comment:
But I think this change is not needed:
```suggestion
```

Reply:
Ok thanks for suggestion!

Reply:
@joeyouss  Can you click the `Commit suggestion` to commit my suggestion.  Thanks.

Reply:
Sure I will do that! Thanks for being helpful. It is great for beginners like me. Appreciate it!